airtouch4pyapi/airtouch.py

```diff
@@ -349,17 +484,130 @@ def DecodeAirtouchMessage(self, payload, map, isGroupBased, chunkSize):
                     #read out entry value from map
                     #run translate on class matching entry name with entry value
                     #set property of entry name on the group response
+    
```

Comment:
the demo script assumes Ac Name is available and thats hard - AirTouch4 will need something like this line inserted *after* line 456 in this file in order to make it compatible:

                resultObject.AcName = "AC " + str(acNumber)

Reply:
Updated!